extensions/pyRevitTools.extension/pyRevit.tab/Selection.panel/select.stack/Select.pulldown/Select Element Types.pushbutton/script.py

```diff
@@ -25,9 +25,11 @@
 all_options.extend(['Fill Patterns', 'Area Schemes', 'Line Styles'])
 selected_option = \
     forms.CommandSwitchWindow.show(all_options,
-                                   message='Pick type category:')
+                                   message='Pick type category:',
+                                   recognize_access_key=False)
```

Comment:
I do not know why it didn't use default `RECOGNIZE_ACCESS_KEY` from `forms`. Maybe you can figure out why. Now I've just put it here